INSTALL.md

```diff
@@ -195,3 +200,69 @@ For example, in a Linux platform, assuming one wishes to execute the `adaboost_d
 ```shell
 ./_cmake_results/intel_intel64_so/adaboost_dense_batch
 ```
+
+## Installation with a conda environment
```

Comment:
Let's rename to something like: `Conda Development Environment Setup`, and remove the last part about running `make` that duplicates the info in the previous chapter.

Reply:
Renamed.